src/varnam_state.cpp

```diff
@@ -388,9 +392,16 @@ void VarnamState::commitText(const FcitxKeySym &key) {
         candidates->candidate(candidateSelected).text().toStringForCommit());
   }
 
-  if (isWordBreak(key)) {
-    stringToCommit = stringutils::concat(stringToCommit, getWordBreakChar(key));
-  }
+   if (isWordBreakKey) {
```

Comment:
I think its better we add another config flag to enable/disable Indic punctuation marks, and keep the default value as false.

Reply:
some people use English punctuation marks in typing their language even language have punctuation marks. we have to pass through punctuations with fcitx config.

Reply:
with new update punctuations also can be enabled or disabled